### DISCUSSION TITLE
全体のお知らせに Qiita Blog の更新も載せてほしい

### DISCUSSION THREAD

**Discussion by uhooi** [increments/qiita-discussions#7] [要望・アイデア・不具合]
Qiita Blog の更新に気づかないことが多く、私は楽しみにしているので、Qiita のお知らせに載ると嬉しいです。

**Comment by kskwtnk** [comment]
@uhooi 

Qiita Blogの更新、楽しみにしていただいてとても嬉しいです！

https://github.com/qiita-discussions/qiita-discussions/discussions/5 にも挙がっていますが、通知欄に載せる情報を整理してみなさんに分かりやすくしたいと思っています。

通知機能としてBlogをお知らせするかは確約できずすみませんが、運営からの情報発信をするためにBlogが更新されたことをもっとキャッチしやすいようにしていきたいと思っています！
